CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+Marcos Cleison Silva Santana
+Ben Albrecht
+Krishna Kumar Dey
```

Comment:
Shouldn't this include Souris Ash with the addition of this PR?

Reply:
Sure! xD